site/src/components/SSHButton/SSHButton.tsx

```diff
@@ -0,0 +1,102 @@
+import Button from "@material-ui/core/Button"
+import Popover from "@material-ui/core/Popover"
+import { makeStyles } from "@material-ui/core/styles"
+import CloudIcon from "@material-ui/icons/CloudOutlined"
+import { useRef, useState } from "react"
+import { CodeExample } from "../CodeExample/CodeExample"
+import { Stack } from "../Stack/Stack"
+import { HelpTooltipLink, HelpTooltipLinksGroup, HelpTooltipText } from "../Tooltips/HelpTooltip"
+
+export interface SSHButtonProps {
+  workspaceName: string
+  agentName: string
+  defaultIsOpen?: boolean
+}
+
+export const SSHButton: React.FC<SSHButtonProps> = ({
+  workspaceName,
+  agentName,
+  defaultIsOpen = false,
+}) => {
+  const anchorRef = useRef<HTMLButtonElement>(null)
+  const [isOpen, setIsOpen] = useState(defaultIsOpen)
+  const id = isOpen ? "schedule-popover" : undefined
+  const styles = useStyles()
+
+  const onClose = () => {
+    setIsOpen(false)
+  }
+
+  return (
+    <>
+      <Button
+        startIcon={<CloudIcon />}
+        size="small"
+        ref={anchorRef}
+        onClick={() => {
+          setIsOpen(true)
+        }}
+      >
+        SSH
+      </Button>
+      <Popover
+        classes={{ paper: styles.popoverPaper }}
+        id={id}
+        open={isOpen}
+        anchorEl={anchorRef.current}
+        onClose={onClose}
+        anchorOrigin={{
+          vertical: "bottom",
+          horizontal: "left",
+        }}
+        transformOrigin={{
+          vertical: "top",
+          horizontal: "left",
+        }}
+      >
+        <HelpTooltipText>
+          To connect with SSH you have to run the following commands:
```

Comment:
```suggestion
          Run the following commands to connect with SSH:
```
I think this is a little more clear and direct